CHANGELOG.md

```diff
@@ -8,6 +8,9 @@ And please only add new entries to the top of this list, right below the `# Unre
 
 # Unreleased
 
+- Make iOS windows usable from other threads.
+- Refactor web backend thread safety slightly.
```

Comment:
I think we tend to not document internal changes if they don't affect the user. Not sure though.